scalanet/src/io/iohk/scalanet/peergroup/DTLSPeerGroup.scala

```diff
@@ -34,6 +34,8 @@ class DTLSPeerGroup[M](val config: Config)(
   private val serverConnector = createServerConnector()
 
   private val channelSubject = PublishSubject[ServerEvent[InetMultiAddress, M]]()
```

Comment:
Can we remove the duplication of `channelSubject` and `connectableObservable` in UDP, DTLS, TCP, TLS. (see for eg https://github.com/input-output-hk/scalanet/blob/A-791/Subjects/scalanet/src/io/iohk/scalanet/monix_subjects/CacheUntilConnectSubject.scala)